cli/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-detection-pipeline.yaml

```diff
@@ -26,7 +26,7 @@ inputs:
   # model_name: yolof_r50_c5_8x8_1x_coco
   # # model - specify the foundation model available in the azureml system registry
   mlflow_model: 
-    path: azureml://registries/azureml/models/yolof_r50_c5_8x8_1x_coco/versions/3
+    path: azureml://registries/azureml/models/mmd-3x-yolof_r50_c5_8x8_1x_coco/versions/8
     type: mlflow_model
```

Comment:
Please make changes in cli\foundation-models\system\finetune\image-object-detection\readme.md as well.

Reply:
Updated.